update_benchmark_fileds.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+###################################################################
+# Sample script that updates fields for codespeed_benchmark table  #
+###################################################################
+import sqlite3
+
+lessisbetter_name = [
+"rescaleHeap.RESCALE_IN",
+"rescaleHeap.RESCALE_OUT",
+"rescaleRocksDB.RESCALE_IN",
+"rescaleRocksDB.RESCALE_OUT",
+"rescaleRocksDB.RESTORE",
+"networkLatency1to1",
+"deployDownstreamTasks.BATCH",
+"deployAllTasks.STREAMING",
+"createScheduler.BATCH",
+"createScheduler.STREAMING",
+"startScheduling.BATCH",
+"startScheduling.STREAMING",
+"calculateRegionToRestart.BATCH",
+"calculateRegionToRestart.STREAMING",
+"partitionRelease.BATCH",
+"initSchedulingStrategy.BATCH",
+"initSchedulingStrategy.STREAMING",
+"schedulingDownstreamTasks.BATCH",
+"buildTopology.BATCH",
+"buildTopology.STREAMING"
+]
+
+if __name__ == "__main__":
+    conn = sqlite3.connect('data.db')
+    c = conn.cursor()
+    for name in lessisbetter_name:
+        sql = "update codespeed_benchmark set lessisbetter=?,units=? where name=?;"
+        cursor = c.execute(sql, (1, 'ms/op', name))
+        conn.commit()
```

Comment:
Not sure if this script (`update_benchmark_fileds.py`) should be added to git because it was a one-off task IIUC.
But if it should, then its purpose and some instructions on how to use it would be great.

Reply:
👍Agreed, when `save_jmh_result.py` changed, this script is no longer needed, deleted it.